src/main/java/de/tum/in/www1/artemis/web/rest/StudentQuestionResource.java

```diff
@@ -179,7 +184,7 @@ public StudentQuestionResource(StudentQuestionRepository studentQuestionReposito
 
     /**
      *
-     * GET /lectures/{lectureId}/student-questions : get all student questions for exercise.
+     * GET /lectures/{lectureId}/student-questions : get all student questions for lecture.
      * @param lectureId the lecture that the student questions belong to
      * @return the ResponseEntity with status 200 (OK) and with body all student questions for exercise
```

Comment:
This still says `exercise`, is that correct?

Reply:
No, I changed it to `lecture` since it is the endpoint to get the questions for a lecture. Must have been forgotten when copy-pasting.